src/modules/keyboardmanager/ui/EditShortcutsWindow.cpp

```diff
@@ -171,9 +172,11 @@ void createEditShortcutsWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMa
 
     // Apply button
     Button applyButton;
-    applyButton.Content(winrt::box_value(L"Apply"));
-    header.SetAlignRightWithPanel(applyButton, true);
-    header.SetLeftOf(cancelButton, applyButton);
+    applyButton.Content(winrt::box_value(L"OK"));
+    applyButton.Background(winrt::Windows::UI::Xaml::Media::SolidColorBrush{ WindowsColors::get_accent_color() });
+    applyButton.Foreground(winrt::Windows::UI::Xaml::Media::SolidColorBrush{ winrt::Windows::UI::Colors::White() });
+    header.SetAlignRightWithPanel(cancelButton, true);
+    header.SetLeftOf(applyButton, cancelButton);
```

Comment:
We should also do the same saving into a copy before saving into KeyboardManagerState logic here. But I think it's better to do that for Shortcuts when implementing #2815, rather than in this PR.